game/static/game/css/board.css

```
@@ -33,6 +33,60 @@ body {
+    font-weight: 800;
+}
+
+.streak-copy {
+    display: flex;
+    flex-direction: column;
+    line-height: 1.1;
+}
+
+.streak-value {
+    color: #f0c040;
+    font-family: 'Consolas', 'Courier New', monospace;
```

Stylelint (17.11.0)

[error] Expected no quotes around "Consolas" (font-family-name-quotes)

(font-family-name-quotes)